cmake/Modules/Findpqxx.cmake

```diff
@@ -15,27 +15,31 @@ set(URL https://github.com/jtv/libpqxx.git)
 set(VERSION 5b17abce5ac2b1a2f8278718405b7ade8bb30ae9)
 set_target_description(pqxx "C++ bindings for postgres client library" ${URL} ${VERSION})
 
+iroha_get_lib_name(PQXX pqxx STATIC)
+
 if (NOT pqxx_FOUND)
   externalproject_add(jtv_libpqxx
       GIT_REPOSITORY ${URL}
       GIT_TAG        ${VERSION}
-      CONFIGURE_COMMAND ${CMAKE_COMMAND} -E env CXXFLAGS=${CMAKE_CXX_FLAGS} CPPFLAGS=-I${postgres_INCLUDE_DIR} PG_CONFIG=${pg_config_EXECUTABLE} ./configure --disable-documentation --with-postgres-include=${pq_INCLUDE_DIR} --with-postgres-lib=${pq_INCLUDE_DIR}
+      CONFIGURE_COMMAND
+          ${CMAKE_COMMAND} -E env CXXFLAGS=${CMAKE_CXX_FLAGS} CPPFLAGS=-I${postgres_INCLUDE_DIR} PG_CONFIG=${pg_config_EXECUTABLE} ./configure --disable-documentation --with-postgres-include=${pq_INCLUDE_DIR} --with-postgres-lib=${pq_INCLUDE_DIR}
```

Comment:
Not really good idea to pass all of the flags, e.g: if I'd like to use sanitizers I don't want more performance issues